### DISCUSSION TITLE
Import BPMN swimlane diagram from text or file

### DISCUSSION THREAD

**Discussion by sebpollmanns** [jgraph/drawio#4245] [General]
Hello everyone. This is a great forum and a awesome tool. Thanks for that.

Is there a way, to import BPMN swimlane-diagrams from text? I found the documentation for all sorts of other diagrams but not BPMN.

I work as a consultant and would love to set up a workflow, where I can talk to my clients about their business processes and record the meeting. Afterwards I want to analyze the transcript with ChatGPT/Copilot and have AI write the code for the BPMN-diagram. 
So far it works amazing with PlantUML code and saves so much time. The next level now would be, to get it working with BPMN-swimlanes.

Has anybode experience with this?
Thanks ahead.
Seb

**Comment by burguetjf** [comment]
Hi Seb, have you read this article ?  https://www.linkedin.com/pulse/revolutionizing-process-modeling-my-journey-custom-gpt-rogotowicz-luc8c/

**Reply by sebpollmanns** [reply]
Great, thank`s for sharing.

**Reply by burguetjf** [reply]
Hi Seb, any luck so far ?  Have you read this other approach :  https://www.linkedin.com/pulse/i-hired-chatgpt-help-me-pushing-human-centric-business-martin-holling/

Please let us know if you made any progress, thanks.

JF

**Reply by sebpollmanns** [reply]
Thank`s so much for sharing. New AI topics keep popping up on my "Must Check"-List. I hope to be able to get to the BPMN-Topic by the end of next months. I will definately have a look at the LinkedIn source and will let you know, if I managed to take it a step further.
Seb

**Comment by burguetjf** [comment]
Then of course there is "BPMN Sketch Miner" (https://www.bpmn-sketch-miner.ai/) which is worth trying !

**Comment by AnastasiiaSolenova** [comment]
Hi, 

I've also been very interested in the topic described in your request lately. Could you please let me know if you have any insights on this topic or any useful resources?

Thank you in advance.
